src/main/kotlin/no/nav/familie/ba/sak/kjerne/brev/BrevUtil.kt

```diff
@@ -271,3 +307,96 @@ fun hentVirkningstidspunktForDødsfallbrev(
 fun hentForvaltningsloverHjemler(vedtakKorrigertHjemmelSkalMedIBrev: Boolean): List<String> {
     return if (vedtakKorrigertHjemmelSkalMedIBrev) listOf("35") else emptyList()
 }
+
+fun skalHenteUtbetalingerEøs(
+    endringstidspunkt: LocalDate,
+    valutakurser: List<Valutakurs>,
+): Boolean =
+    valutakurser.tilSeparateTidslinjerForBarna().mapValues { (_, valutakursTidslinjeForBarn) -> valutakursTidslinjeForBarn.beskjærFraOgMed(endringstidspunkt.tilMånedTidspunkt()) }.filterValues { it.erIkkeTom() }.isNotEmpty()
```

Comment:
Tror det kan være ryddig å splitte opp denne litt. 

Synes også det blir vanskelig med dobbel not i `.filter.{it.erIkkeTom}.isNotEmpty()`
```suggestion
fun skalHenteUtbetalingerEøs(
    endringstidspunkt: LocalDate,
    valutakurser: List<Valutakurs>,
): Boolean {
    val valutakurserEtterEndringstidspunktetTidslinjer = valutakurser.tilSeparateTidslinjerForBarna().mapValues { (_, valutakursTidslinjeForBarn) -> valutakursTidslinjeForBarn.beskjærFraOgMed(endringstidspunkt.tilMånedTidspunkt()) }

    return valutakurserEtterEndringstidspunktetTidslinjer.any { it.value.erIkkeTom() }
}
```